src/configs/env.ts

```diff
@@ -80,6 +84,11 @@ export const ENABLE_IRS_PERFORMANCE_REPORT =
   process.env.REACT_APP_ENABLE_IRS_PERFORMANCE_REPORT === 'true';
 export type ENABLE_IRS_PERFORMANCE_REPORT = typeof ENABLE_IRS_PERFORMANCE_REPORT;
 
+/** Do you want to enable IRS Lite Performance Reporting */
+export const ENABLE_IRS_LITE_PERFORMANCE_REPORT =
+  process.env.REACT_APP_ENABLE_IRS_LITE_PERFORMANCE_REPORT === 'true';
+export type ENABLE_IRS_LITE_PERFORMANCE_REPORT = typeof ENABLE_IRS_LITE_PERFORMANCE_REPORT;
+
```

Comment:
This may not be needed since we don't have IRS lite performance report. Right ?

Reply:
Done